lib/docker/executeDockerBuild.ts

```diff
@@ -40,45 +41,69 @@ import * as os from "os";
 import * as path from "path";
 import { cleanImageName } from "./name";
 
-/**
- * Options to configure the Docker image build
- */
-export interface DockerOptions {
-
+export interface DockerRegistry {
     /**
-     * Provide the image tag for the docker image to build
+     * Push Url for this registry
      */
-    dockerImageNameCreator?: DockerImageNameCreator;
+    url: string;
```

Comment:
This is not a url. Please change back to registry. Eg. `atomist` is a valid registry but that is commonly not referred to as url. Alternatively docker login refers to this as `server`

Reply:
https://github.com/atomist/sdm-pack-docker/blob/ipcrm_multi_registry/lib/docker/executeDockerBuild.ts#L88

dockerImageNameCreator is still defined; this is part of a new interface defined above it.  https://github.com/atomist/sdm-pack-docker/blob/ipcrm_multi_registry/lib/docker/executeDockerBuild.ts#L44-L79